docs/installation/installation.md

```diff
@@ -39,6 +39,8 @@ Select the Liberty tools software and click `Next`.
 
 ### Avoid trouble
 
-To fully setup your environment, be sure to check that your local PATH environment variable contains the paths to your Docker, Maven, and Gradle installations. Be sure that the PATH is also visible to the terminal in your Eclipse IDE.
+The plugin uses the available Java, Maven, Gradle, and Docker installations to run the command actions associated with each project on the dashboard. More precisely, the plugin detects what Java installation to use by first checking the `JAVA_HOME` environment variable. If that variable is not set, it checks for the `java.home` system property. Similarly, the plugin detects what Maven/Gradle installation to use by checking the Maven and Gradle installations defined on the PATH environment variable; however, if the project defines a Maven/Gradle wrapper, the wrapper is used to run the command actions instead. Last, the plugin detects what Docker installation to use by checking the Docker installation defined on the PATH environment variable.
```

Comment:
If that variable is not set, it checks for the `java.home` system property.
 =>
If that variable is not set, the JRE used to run the Eclipse IDE itself is used.